Clappr.xcodeproj/xcshareddata/xcschemes/Clappr_Tests.xcscheme

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Maybe we can avoid adding this file or is there a special reason for this?

Reply:
this change is about showing test target on xcode menu. IMHO it helps. But I can revert it if you want to.

Reply:
Okey-Dokey.
=]